src/managers/PointerManager.cpp

```diff
@@ -635,7 +635,14 @@ Vector2D CPointerManager::closestValid(const Vector2D& pos) {
         float    distanceSq = __FLT_MAX__;
 
         for (auto& b : currentMonitorLayout.monitorBoxes) {
-            auto p      = b.closestPoint(vec);
+            auto p = b.closestPoint(vec);
+
+            auto max_x = std::nextafter(b.x + b.w, b.x + b.w - 1);
```

Comment:
wth is nextafter?

Reply:
"Returns the next representable value of `from` in the direction of `to`. "
So in this case returns the prev representable value from `x + width`.
If `x + width` equals `3840` like the 4k display, the prev representable by `double` type is `3839.9999999999995453...`, because floating point precision and consistency is a joke, and is for this that i'm worried about using floating point values for cursor position

Reply:
round exists